concepticondata/conceptlists/RonaTas-2011-431.tsv

```diff
@@ -0,0 +1,432 @@
+RonaTas-2011-431-82	82	club, mace, reed-mace	1763	CLUB
+RonaTas-2011-431-83	83	beech	1950	BEECH
+RonaTas-2011-431-84	84	sin	2971	SIN
+RonaTas-2011-431-85	85	(blunt) end of something, a rod with an end cut, obtuse end of a field-strip, the shorter side or end of a piece of furniture, foot of the hill		
+RonaTas-2011-431-86	86	magical		
+RonaTas-2011-431-87	87	a kind of willow, Salix elægnos		
+RonaTas-2011-431-88	88	trap; snare (for catching birds); bird cage	1075	TRAP (PITFALL)
+RonaTas-2011-431-89	89	a kind of willow, used for baskets and in viticulture, Salix purpurea		
+RonaTas-2011-431-90	90	a kind of willow		
+RonaTas-2011-431-91	91	a kind of wolfs milk, Easter flower, Euphorbia		
+RonaTas-2011-431-92	92	a fish, Leuciscus souffia agassizi	227	FISH
+RonaTas-2011-431-93	93	only, just	1580	ONLY
+RonaTas-2011-431-94	94	pick-axe; mace; a small axe; war hammer		
+RonaTas-2011-431-95	95	nettle, Urtica	33	NETTLE
+RonaTas-2011-431-96	96	thicket, brushwood, dry foliage, grove, meadow	2819	GRASSLAND
+RonaTas-2011-431-97	97	thicket, brushwood; dry foliage, grove, meadow	2819	GRASSLAND
+RonaTas-2011-431-98	98	(silver) cup, wooden ladle, earthenware bowl	476	CUP
+RonaTas-2011-431-99	99	fighting, troop, army group, battle, quarrel, hubbub, flock	1848	QUARREL
+RonaTas-2011-431-100	100	to add, to join; to buckle up something	1743	JOIN
+RonaTas-2011-431-101	101	clasp		
+RonaTas-2011-431-102	102	measle, smallpox, pocks; skin eruption; decorative embroidery	1054	SMALLPOX
+RonaTas-2011-431-103	103	to bind (something)	1917	TIE
+RonaTas-2011-431-104	104	impurities, tow, chaff	2363	CHAFF (HUSK)
+RonaTas-2011-431-105	105	snail, Helix; snail-shell, pulley, vertebra	1543	SNAIL
+RonaTas-2011-431-106	106	a wine of low quality, a fruit wine		
+RonaTas-2011-431-107	107	secretion, mucus discharged from the eyes		
+RonaTas-2011-431-108	108	squint-eyed, blind in one eye, onearmed, one-legged	87	SQUINT-EYED
+RonaTas-2011-431-109	109	a carpenters axe with a long helve		
+RonaTas-2011-431-110	110	stallion	936	STALLION
+RonaTas-2011-431-111	111	sexual organ of large male animals, of a bull; a whip made from it		
+RonaTas-2011-431-112	112	feast, christening feast (at baptism)		
+RonaTas-2011-431-113	113	to become smaller, remain small, to sway, to totter, tosprain	573	SWAY
+RonaTas-2011-431-114	114	stubborn	3634	STUBBORN
+RonaTas-2011-431-115	115	field-guard; herald; Turkish envoy		
+RonaTas-2011-431-116	116	rough, coarse; ugly; wild, unbridled	1923	ROUGH
+RonaTas-2011-431-117	117	to wind, to turn, to distrort, misinterpret, to spin a yarn	1588	TURN
+RonaTas-2011-431-118	118	bobbin, spindle	277	SPINDLE
+RonaTas-2011-431-119	119	excellent; worthless		
+RonaTas-2011-431-120	120	a short fur coat		
+RonaTas-2011-431-121	121	grist, groats, soft hail		
+RonaTas-2011-431-122	122	noon, south	12	MIDDAY
+RonaTas-2011-431-123	123	nut; walnut, Juglans	2042	WALNUT
+RonaTas-2011-431-124	124	pig, Sus domestica	1337	PIG
+RonaTas-2011-431-125	125	to lean, topple over, stream down	3619	LEAN
```

Comment:
I'd unmap, as this includes many different meanings.

Reply:
Resolved in 70cb680e6327a35615c7b0841c50828f1bee1534